test/cli/install/semver.test.ts

```diff
@@ -105,6 +105,24 @@ describe("Bun.semver.order()", () => {
     }
   });
 
+  // not supported by semver, but supported by Bun
+  test.each([
+    ["0", "0.0"],
+    ["1", "1.0"],
+    ["1.2", "1.2.0"],
+    ["1.x", "1.0.x"],
+    ["1.x.x", "1.0.x"],
+    ["2.x", "1.x"],
+    ["2.x", "2.1"],
+    ["2", "1"],
```

Comment:
this also applies to wildcards like `*`, `2.*`, `2.1.*`? Could you add a couple tests for those as well?